docs/get-started/quickstarts/detect-people.md

```diff
@@ -61,25 +64,30 @@ For more detailed information, including optional attribute configuration, see t
 {{% /expand%}}
 {{%expand "Step 4: Deploy a person detection model" %}}
 
-Now add an `ML model` service:
+Now add an `ML model` service that can detect a person:
 Click **+**, click **Service** and select the `ML model` type, then select the `TFLite CPU` model.
+Create the service.
```

Comment:
```suggestion
```

ditto here, I think that should be clear from context, or maybe it should say "Now create an `ML model` service that can detect a person"?

Reply:
I see what you mean but you do have to select create so I thought this was still helpful